org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -45,7 +75,249 @@ package lang {
 		method getKey() { this.getX() }
 		method getValue() { this.getY() }
 	}
- 
+	
+	class Collection {
+		method max(closure) = this.absolute(closure, [a,b | a > b])
+		method min(closure) = this.absolute(closure, [a,b | a < b])
+		
+		method absolute(closure, criteria) {
+			val result = this.fold(null, [acc, e|
+				val n = closure.apply(e) 
+				if (acc == null)
+					new Pair(e, n)
+				else {
+					if (criteria.apply(n, acc.getY()))
+						new Pair(e, n)
+					else
+						acc
+				}
+			])
+			return if (result == null) null else result.getX()
+		}
+		 
+		// non-native methods
```

Comment:
I think this comment is wrong because there are more non-native methods before it